<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AchievementSettingsWidget</class>
 <widget class="QWidget" name="AchievementSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>524</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="settingsLayout">
      <item row="1" column="1">
       <widget class="QCheckBox" name="spectatorMode">
        <property name="text">
         <string>Enable Spectator Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="enable">
        <property name="text">
         <string>Enable Achievements</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="unofficialAchievements">
        <property name="text">
         <string>Test Unofficial Achievements</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="encoreMode">
        <property name="text">
         <string>Enable Encore Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="hardcoreMode">
        <property name="text">
         <string>Enable Hardcore Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="useRAIntegration">
        <property name="text">
         <string>Enable RAIntegration (Development Only)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notificationBox">
     <property name="title">
      <string>Notifications</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="1" column="0">
       <widget class="QCheckBox" name="leaderboardNotifications">
        <property name="text">
         <string>Show Leaderboard Notifications</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QSlider" name="achievementNotificationsDuration">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="achievementNotificationsDurationLabel">
          <property name="text">
           <string>5 seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="achievementNotifications">
        <property name="text">
         <string>Show Achievement Notifications</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <widget class="QSlider" name="leaderboardNotificationsDuration">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="leaderboardNotificationsDurationLabel">
          <property name="text">
           <string>5 seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="soundEffects">
        <property name="text">
         <string>Enable Sound Effects</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="leaderboardTrackers">
        <property name="text">
         <string>Show Leaderboard Trackers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progress Tracking</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="QLabel" name="challengeIndicatorModeLabel">
        <property name="text">
         <string>Challenge Indicators:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="progressIndicators">
        <property name="text">
         <string>Show Progress Indicators</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="challengeIndicatorMode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loginBox">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
      <item>
       <widget class="QLabel" name="loginStatus">
        <property name="text">
         <string>Username:
Login token generated at:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshProgress">
        <property name="text">
         <string>Refresh Progress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewProfile">
        <property name="text">
         <string>View Profile...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Login...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gameInfoBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>75</height>
      </size>
     </property>
     <property name="title">
      <string>Game Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="gameInfo">
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
